authors/apps/authentication/tests/test_views.py

```diff
@@ -115,5 +113,18 @@ def test_invalid_length_of_token(self):
             format='json')
         self.assertEqual(response.status_code, 403)
 
-
-
+    def test_rest_password_email(self):
+        user = User.objects.create_user(username='sprintersspace',
+                                        password='sprintersspacePassword', email='sprintersspace@gmail.com')
+        user_reset_email = {"email": "sprintersspace@gmail.com"}
+        key1 = 'message'
+        value1 = 'check your email for the rest password token'
+        key2 = 'status_code'
+        value2 = '200'
+        response = self.client.post(reverse('auth:password_reset'),
+                                    content_type='application/json', data=json.dumps(user_reset_email))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data[2:9], key1)
```

Comment:
Why the choice to use slices for test Assertion?
This usually brings up errors since you will be trying to hash a mutable object